static/test/javascripts/spec/common/onward/breaking-news.spec.js

```diff
@@ -52,13 +52,16 @@ define([
                         webTitle: 'Breaking News',
                         collections: collections
                     })]);
-                    breakingNews().then(function(result) {
+                    Promise.resolve().then(function () {
```

Comment:
Out of interest, is there a reason why you have to coerce the exception into a rejected promise? Is it so that you can only have one `catch`, instead of two (like below)?

``` js
try {
  breakingNews().catch(reject);
} catch (e) {
  reject(e);
}
```

Related, I found this article interesting (which also documents your `Promise.resolve` solution): [Promise-based functions should not throw exceptions](http://www.2ality.com/2016/03/promise-rejections-vs-exceptions.html)